core/components/molecules/emptyState/EmptyStateDescription.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+import { BaseProps } from '@/utils/types';
+import { Text } from '@/index';
+import { TextSize } from 'types';
+import EmptyStateContext from './EmptyStateContext';
+
+export interface EmptyDescriptionProps extends BaseProps {
+  /**
+   * The content to be displayed within the component
+   * Can be any renderable React Text.
+   */
+  children: React.ReactText;
+}
+
+const EmptyStateDescription = (props: EmptyDescriptionProps) => {
+  const { children, className, ...rest } = props;
+  const contextProp = React.useContext(EmptyStateContext);
+
+  const { size = 'standard', maxWidth } = contextProp;
+
+  const textSize: Record<string, TextSize> = {
+    standard: 'regular',
+    compressed: 'regular',
+    tight: 'small',
+  };
+
+  const descriptionClasses = classNames(
+    {
+      [`EmptyState-text`]: true,
+    },
+    className
+  );
+
+  return (
+    <Text
+      size={textSize[size]}
+      style={{ maxWidth: maxWidth }}
+      appearance="subtle"
+      className={`mt-3 ${descriptionClasses}`}
```

Comment:
Please add `mt-3` class inside existing `EmptyState-text` class

Reply:
@anuradha9712 i am using the same class in EmptyStateTitle.tsx as they both have the same property they only difference is this mt-3 do i need to make a separate class for this or it will be fine to use this here like this?

Reply:
Yes, please make a separate class for different component or else you can add `mt-3` in descriptionClass as well like `['mt-3']: true` in addition of `EmptyState-text` class

Reply:
done